src/test/resources/log4j.properties

```diff
@@ -0,0 +1,9 @@
+log4j.rootCategory=WARN, console
+log4j.appender.console=org.apache.log4j.ConsoleAppender
+log4j.appender.console.target=System.err
+log4j.appender.console.layout=org.apache.log4j.PatternLayout
+log4j.appender.console.layout.ConversionPattern=%d{yy/MM/dd HH:mm:ss} %p %c{1}: %m%n
+
+log4j.logger.org.eclipse.jetty.util.component.AbstractLifeCycle=ERROR
+log4j.logger.org.apache.spark.repl.SparkIMain$exprTyper=INFO
```

Comment:
Is this the reason that the Python tests are logging at INFO level?  It would be nice to make them log at WARN level to be more readable (and then you could probably remove that hack above to limit the size of Travis logs).